inc/thorvg.h

```diff
@@ -212,7 +212,7 @@ enum class BlendMethod : uint8_t
  */
 enum class CanvasEngine : uint8_t
 {
-    All = 0,       ///< All feasible rasterizers.
+    All = 0,       ///< All feasible rasterizers. @since 1.0
     Sw = (1 << 1), ///< CPU rasterizer.
     Gl = (1 << 2), ///< OpenGL rasterizer.
```

Comment:
does GL backend needs to be marked as experimetal API ?

Reply:
@RuiwenTang It's been already there from the beginning. Instead, we should not change the enum value.  GlCanvas has that experimental tag so It think it wouldn't be a problem.

Reply:
@RuiwenTang fyi, "All" was newly added since 1.11.